internal/commands/app/create_inputs.go

```diff
@@ -153,56 +154,69 @@ func (i *createInputs) resolveLocalPath(ui terminal.UI, wd string) (string, erro
 	return fullPath, nil
 }
 
-func (i *createInputs) resolveCluster(client atlas.Client, groupID string) (dataSourceCluster, error) {
+func (i *createInputs) resolveClusters(client atlas.Client, groupID string) ([]dataSourceCluster, error) {
 	clusters, err := client.Clusters(groupID)
 	if err != nil {
-		return dataSourceCluster{}, err
+		return []dataSourceCluster{}, err
 	}
-	var clusterName string
-	for _, cluster := range clusters {
-		if i.Cluster == cluster.Name {
-			clusterName = cluster.Name
-			break
-		}
+
+	existingClusters := map[string]atlas.Cluster{}
+	for _, c := range clusters {
+		existingClusters[c.Name] = c
 	}
-	if clusterName == "" {
-		return dataSourceCluster{}, errors.New("failed to find Atlas cluster")
+
+	var dsClusters []dataSourceCluster
+	for _, clusterName := range i.Clusters {
+		if _, ok := existingClusters[clusterName]; !ok {
+			continue
+		}
+		dsClusters = append(dsClusters,
+			dataSourceCluster{
+				Name: "mongodb-atlas",
+				Type: "mongodb-atlas",
```

Comment:
we should probably move this to a `const` somewhere, I'm thinking probably in `realm` package...

Reply:
at the end of this file maybe? https://github.com/10gen/realm-cli/blob/362361d1ee6a774e88a3451237ec8dc9ec1332b6/internal/cloud/realm/realm.go

Reply:
that could work, or just thinking forward a little I think creating a `services.go` file wouldn't be the worst thing here either (we have a similarly small `auth_providers.json` file)